chat.iamb.iamb.metainfo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>chat.iamb.iamb</id>
+
+  <name>iamb</name>
+  <summary>A terminal Matrix client for Vim addicts</summary>
+  <url type="homepage">https://iamb.chat</url>
+
+  <releases>
+    <release version="0.0.9" date="2024-03-28"/>
+  </releases>
+
+  <developer_name>Ulyssa</developer_name>
+  <update_contact>git@ulyssa.dev</update_contact>
+
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://iamb.chat/static/images/iamb-demo.gif</image>
```

Comment:
gif won't likely work on the website can you png/jpgs?

Reply:
Updated in ulyssa/iamb#335 to point at [this PNG](https://iamb.chat/static/images/metainfo-screenshot.png).